test/RelayServer.test.ts

```diff
@@ -1242,6 +1260,51 @@ contract('RelayServer', function (accounts) {
     // })
   })
 
+  describe('Alerted state as griefing mitigation', function () {
+    let newServer: RelayServer
+    beforeEach('should enter an alerted state for a configured blocks delay after paymaster rejecting an on-chain tx', async function () {
+      id = (await snapshot()).result
+      newServer = await bringUpNewRelay({ alertedBlockDelay: 100 })
+      await newServer._worker(await _web3.eth.getBlock('latest'))
+      await attackTheServer(newServer)
+    })
+    afterEach(async function () {
+      await revert(id)
+    })
+
+    async function attackTheServer (server: RelayServer): Promise<void> {
+      const _sendTransactionOrig = server._sendTransaction
+      const _sendTransaction = async function ({ signer, method, destination, value = '0x', gasLimit, gasPrice }: SendTransactionDetails): Promise<SignedTransactionDetails> {
+        await rejectingPaymaster.setRevertPreRelayCall(true)
+        // @ts-ignore
+        return (await _sendTransactionOrig.call(server, ...arguments))
+      }
+      server._sendTransaction = _sendTransaction
+      await relayTransaction(server, options, { paymaster: rejectingPaymaster.address }, false)
+      const currentBlock = await _web3.eth.getBlock('latest')
+      await server._worker(currentBlock)
+      assert.isTrue(server.alerted, 'server not alerted')
+      assert.equal(server.alertedBlock, currentBlock.number, 'server alerted block incorrect')
+    }
+
+    it('delay txs in alerted state', async function () {
```

Comment:
should; transactions;

no need to omit letters really

Reply:
fixed